VirtoCommerce.MarketingModule.Web/Scripts/promotion/blades/catalog-entry-list.tpl.html

```diff
@@ -0,0 +1,47 @@
+<div class="blade-content __medium-wide">
+    <div class="blade-inner">
+        <div class="inner-block">
+            <div class="table-wrapper" ng-init="setGridOptions('catalogConditionsGrid', {
+                data: objects,
+                useExternalSorting: true,
+                rowTemplate: 'catalog-conditions-item-list-row.tpl.html',
+                rowHeight: 48,
+                columnDefs: [
+                    { name: 'imgSrc', displayName: 'catalog.blades.catalog-items-select.labels.picture', cellTemplate: 'catalog-conditions-item-list-image-cell.tpl.html', enableColumnResizing: false, enableSorting: false, width: 60 },
+                    { name: 'name', displayName: 'catalog.blades.catalog-items-select.labels.name', enableSorting: false },
+                    { name: 'code', displayName: 'catalog.blades.catalog-items-select.labels.sku', width: 170, cellTemplate: 'catalog-conditions-item-list-SKU.cell.html', enableSorting: false }]})">
+                <div ui-grid="gridOptions" ui-grid-pinning ui-grid-auto-resize ui-grid-save-state ui-grid-selection ui-grid-resize-columns ui-grid-move-columns ui-grid-height></div>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="blade-static __bottom">
```

Comment:
Need to use default Ok/Cancel widget instead https://github.com/VirtoCommerce/vc-module-store/blob/dev/VirtoCommerce.StoreModule.Web/Scripts/blades/taxProvider-detail.tpl.html#L1